data/block/blockV2.go

```diff
@@ -468,6 +468,11 @@ func (hv2 *HeaderV2) SetEpochStartMetaHash(hash []byte) error {
 	return hv2.Header.SetEpochStartMetaHash(hash)
 }
 
+// HasScheduledSupport returns true if the header version supports scheduled data
```

Comment:
HasScheduledSupport returns true as the second block version does support scheduled data

Reply:
done